src/dialogs/config-flow/step-flow-abort.ts

```diff
@@ -33,6 +53,36 @@ class StepFlowAbort extends LitElement {
     `;
   }
 
+  private async _handleMissingCreds() {
+    if (
+      !(await showConfirmationDialog(this, {
+        title: this.hass.localize(
+          "ui.panel.config.integrations.config_flow.missing_credentials",
+          {
+            integration: domainToName(this.hass.localize, this.domain),
+          }
+        ),
+      }))
+    ) {
+      this._flowDone();
```

Comment:
Why not always call `flowDone` ?

Reply:
Fixed, now based on other thread about the parent dialog.